cli/cmd/install-cni-plugin.go

```diff
@@ -129,6 +143,12 @@ func newCNIInstallOptionsWithDefaults() (*cniPluginOptions, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	cniPluginImage := cniPluginImage{
+		name:    defaultDockerRegistry + "/cni-plugin",
+		version: "v1.0.0",
```

Comment:
WDYT about moving this into a `var` in `version.go` alongside `ProxyInitVersion`?

Reply:
Yeah, that looks better. Thanks!